R/install.R

```diff
@@ -301,7 +301,21 @@ find_hugo = local({
 #' @export
 #' @rdname find_hugo
 remove_hugo = function(version = getOption('blogdown.hugo.version'), force = FALSE) {
-  for (f in find_hugo(version)) {
+  installed = if (interactive()) {
```

Comment:
yes, I used `missing` as if the option is set but the user has not asked for the version (version not provided), it would still show the interactive menu